test-integration/scenarios/loans/1_eth_takeout_loan_and_2_repays_full_successfully.js

```diff
@@ -1,153 +0,0 @@
-// Util classes
```

Comment:
why are we deleting this?

Reply:
We already have this scenario in file ```test-integration/scenarios/loans/1_takeout_loan_and_2_repays_full_successfully.js``` (and it is supporting the multi execution per market)